### PR TITLE
add ut for binding

### PR DESCRIPTION
**Pull request by AllenZMC**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
add ut for binding
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/util/helper/binding_test.go [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+
 	workv1alpha2 "github.com/karmada-io/karmada/pkg/apis/work/v1alpha2"
 )
 
+const (
+	ClusterMember1 = "member1"
+	ClusterMember2 = "member2"
+	ClusterMember3 = "member3"
+	ClusterMember4 = "member4"
+)
```
Comment (XiShanYongYe-Chang) [inline pkg/util/helper/binding_test.go:17]: Maybe we don't need these constants.

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Thanks~

File: pkg/util/helper/binding_test.go (hunk 1 above)
Comment (AllenZMC) [reply pkg/util/helper/binding_test.go]: fixed

**Review by XiShanYongYe-Chang — COMMENTED** [review]
/lgtm

**Comment by AllenZMC** [comment]
@RainbowMango ra

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**